manifests/image.pp

```diff
@@ -73,22 +73,19 @@
     }
   }
 
-  case $ensure {
-    'present': {
-      exec { "pull_image_${title}":
-        command => "podman image pull ${_flags} ${image}",
-        unless  => "podman image exists ${image}",
-        path    => '/sbin:/usr/sbin:/bin:/usr/bin',
-        *       => $exec_defaults,
-      }
+  if $ensure == 'present' {
+    exec { "pull_image_${title}":
+      command => "podman image pull ${_flags} ${image}",
+      unless  => "podman image exists ${image}",
+      path    => '/sbin:/usr/sbin:/bin:/usr/bin',
+      *       => $exec_defaults,
     }
-    default: {
-      exec { "pull_image_${title}":
-        command => "podman image pull ${_flags} ${image}",
-        unless  => "podman rmi ${image}",
-        path    => '/sbin:/usr/sbin:/bin:/usr/bin',
-        *       => $exec_defaults,
-      }
+  } elsif $ensure == 'absent' {
+    exec { "pull_image_${title}":
+      command => "podman rmi ${_flags} ${image}",
```

Comment:
I don't think passing `_flags` here is a good idea, `podman rmi` only knows about:
```
  -a, --all        Remove all images
  -f, --force      Force Removal of the image
  -i, --ignore     Ignore errors if a specified image does not exist
      --no-prune   Do not remove dangling images
```
but the tests indicate you pass in much more